.gitignore

```diff
@@ -0,0 +1 @@
+tox.ini
```

Comment:
please don't add tox.ini to `.gitignore`: https://github.com/splunk-soar-connectors/lansweeper/pull/1#discussion_r783509354